errors/sync-dynamic-apis.mdx

```diff
@@ -64,6 +64,31 @@ function Page({ params }) {
 }
 ```
 
+### Unmigratable Cases
+
+If Next.js codemod found anything that is not able to be migrated by the codemod, it will leave a comment with `@next-codemod-error` prefix and the suggested action, for example:
```

Comment:
Need to specify the comment prefix in the "good to know" section of the codemod docs as well: https://nextjs.org/docs/canary/app/building-your-application/upgrading/codemods